dcc-storage-client/src/main/java/org/icgc/dcc/storage/client/transport/MemoryMappedParallelPartObjectTransport.java

```diff
@@ -115,13 +115,11 @@ public Part call() throws Exception {
                   log.debug("Fail checksum. Reupload part: {}", part);
                   progress.startTransfer();
                   proxy.uploadPart(channel, part, objectId, uploadId);
-                  progress.incrementBytesWritten(part.getPartSize());
                 }
                 progress.incrementChecksumParts();
               } else {
                 progress.startTransfer();
                 proxy.uploadPart(channel, part, objectId, uploadId);
-                progress.incrementBytesWritten(part.getPartSize());
```

Comment:
I've been regression-testing this part this past little while. It was calling incrementBytesWritten() here as well as in ProgressInputStream/ProgressOutputStream on every read/write call. This seemed like double-counting to me.